AUTHORS

```diff
@@ -1,5 +1,2 @@
-Martin Aigner <martin@maigner.net>
-Christian Barthel <cbarthel@cs.uni-salzburg.at>
-Christoph Kirsch <ck@cs.uni-salzburg.at>
-Michael Lippautz <michael.lippautz@gmail.com>
-Simone Oblasser <simone.oblasser@cs.uni-salzburg.at>
\ No newline at end of file
+Tobias Seiler <tobias.seiler@stud.sbg.ac.at>
```

Comment:
See home work flow description again. _Add_ your names! Do not replace previous authors. That's called plagiarism!